src/odemis/driver/simcam.py

```diff
@@ -155,6 +155,7 @@ def clip_max_res(img_res):
         self._generator = None
         # Convenience event for the user to connect and fire
         self.softwareTrigger = model.Event()
+        self._last_acq_time = 0  # Time of latest acquisition
```

Comment:
This var will be a float isn't it (time.time() is a float).
Should it be initialized as a float also? So 0.0
Its not that important since this is python but it is more consequent.

Reply:
Technically Python still differentiates between floats and ints. However, since Python 2.7, it now behaves in pretty much exactly the same with a float or an int. I like to think of int as a subtype of float (in an object-oriented view), and so 0 is fine. 

So I'd rather keep `0`, it's just easier to read, and not showing any low-level details.